Src/FluentAssertions/AssertionExtensions.cs

```diff
@@ -816,6 +816,31 @@ public static TTo As<TTo>(this object subject)
             return (await task).WithInnerException<TInnerException>(because, becauseArgs);
         }
 
+#pragma warning restore AV1755
```

Comment:
Why restoring warning for the documentation?
Instead I would recommend to disable this warning for the whole file.
All these extension will not follow the rule because of the fluent api.

Reply:
Ouch, this should not be here. Should I amend or just add new commit?

Reply:
Whatever works for you. I follow [this procedure](https://www.continuousimprover.com/2020/03/keep-source-control-history-clean.html).